src/connectors/dispatcher/impl.unit.js

```diff
@@ -201,7 +201,7 @@ describe('dispatcher impl', function () {
                                 account: 654321,
                                 recipient: '9574cba7-b9ce-4725-b392-e959afd3e69a',
                                 correlation_id: '5c9ae28b-1728-4067-b1f3-f4ad992a8296',
-                                url: 'https://cloud.redhat.com/api/remediations/v1/remediations/f376d664-5725-498d-8cf9-bbfaa51b80ca/playbook?hosts=9574cba7-b9ce-4725-b392-e959afd3e69a&localhost',
+                                url: 'https://console.redhat.com/api/remediations/v1/remediations/f376d664-5725-498d-8cf9-bbfaa51b80ca/playbook?hosts=9574cba7-b9ce-4725-b392-e959afd3e69a&localhost',
```

Comment:
AFAIK the API urls are not changing. However, given this is the mock implementation I don't think it matters one way or another.